workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/flows/redeem/ConfidentialRedeemFungibleTokensFlowHandler.kt

```diff
@@ -1,15 +0,0 @@
-package com.r3.corda.lib.tokens.workflows.flows.redeem
-
-import co.paralleluniverse.fibers.Suspendable
-import net.corda.core.flows.FlowLogic
-import net.corda.core.flows.FlowSession
-
-/**
- * Responder flow to [ConfidentialRedeemFungibleTokensFlow].
- */
-class ConfidentialRedeemFungibleTokensFlowHandler(val otherSession: FlowSession) : FlowLogic<Unit>() {
```

Comment:
Hi @adagys. Thanks for the PR . What's the reason for removing this class?

Reply:
It seems to have lost its purpose, currently it just runs a subflow. I made the `ConfidentialRedeemFungibleTokensHandler` call `RedeemTokensFlowHandler` directly.

Reply:
@adagys - the reason we leave these flows in place is due to the nature of this cordapp it's an SDK and so people will usually invoke these handlers as part of their own responder, could we put it back?

Reply:
@roastario All the other `TokenFlowHandler`s just call `RedeemTokensFlowHandler` directly, but here we have `ConfidentialRedeemFungibleTokensHandler -> ConfidentialRedeemFungibleTokensFlowHandler -> RedeemTokensFlowHandler`

People could inline `ConfidentialRedeemFungibleTokensHandler` instead. But if this is considered API breakage I'm happy to put it back.